generators/app/templates/src/app/screens/Login/i18n.js

```diff
@@ -1,13 +1,7 @@
-import { buildTranslations, buildConstants } from '../../../utils/i18nUtils';
+import i18next from 'i18next';
 
-const namespace = 'login';
-const translations = {
-  es: {
-    loginMessage: 'Login, {{0}} !',
-    user: 'Usuario',
-    password: 'Contraseña'
-  }
-};
-
-export const strings = buildConstants(translations, namespace, 'es');
-export default buildTranslations(translations, namespace);
+i18next.addResources('es', 'LOGIN', {
```

Comment:
If we were using an i18n file per screen, maybe we can use the screen route as namespace, what u think? @pabloferro @guidoprinc @sfernandez11 ?

Reply:
@WFolini

I think it would just make the code more complicated.
```js
import Routes ...
i18next.addResources('es', Routes.LOGIN, {
```
```js
placeholder={i18next.t(`${Routes.LOGIN}:PASSWORD`)}
```

If you think that's worth it I can change it.